go.mod

```diff
@@ -10,6 +10,7 @@ require (
 	github.com/gogo/protobuf v1.3.2 // indirect
 	github.com/google/go-cmp v0.5.2
 	github.com/google/uuid v1.1.2
+	github.com/gorilla/mux v1.8.0
```

Comment:
Not really sure we need a third-party library for the trivial path routing we're doing — could use the stdlib http mux and do simple regex or string splitting on the URL to route.

Reply:
Done.